### PR TITLE
[For 2.6.3] Prepare new stable release 2.6.3

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
related to #2788 

## Checkpoints

The goal of this section is to help code reviewer and merge request author to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewer**

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

**For developper**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] Original issue have been updated with a short resume of implemented solution

**For quality team**

- [x] Tested by quality team

### CODE REVIEW

File: backend/doc/setting.md [hunk 1]
```diff
@@ -312,6 +312,9 @@ to enable mail notification, smallest config is this:
 Don't forget to set `website.base_url` and `website.title` for the frontend, as some features use them to
 link the frontend in emails.
 
+:warning: its necessary to check if your smtp configuration working good before using tracim.
```
Comment (raphj) [inline backend/doc/setting.md:315]:
```suggestion
:warning: It is necessary to check if your SMTP configuration is working correctly before using Tracim.
```

File: backend/doc/setting.md [hunk 2]
```diff
@@ -312,6 +312,9 @@ to enable mail notification, smallest config is this:
...
+In next release we purpose a quick solution to test if your smtp configuration work properly.
```
Comment (raphj) [inline backend/doc/setting.md:316]:
```suggestion
In the next release we will include a quick solution to test if your STMP configuration works properly.
```

**Comment by PhilippeAccorsi** [comment]
I don't know if its necessary to explain how delete and restore elasticsearch if using tracim with docker.

Maybe I can add information in issue about elasticsearch directly.

File: changelog.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
 
 
+## 2.6.3 / 2020-04-09
+
+### Improvement
+
+- Database index improvement to increase Tracim reactivity (#2787)
```
Comment (raphj) [inline changelog.md:7]:
```suggestion
- Better Tracim reactivity by improving Database indexing (#2787)
```

File: changelog.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+
+### Fixed issues
+
+- Frontend: #2688
+- Backend: #2687, #2793, #2811, #2812
+
+### Other Changes
+
+- ElasticSearch: Refactor of indexing organization. So its necessary to drop existing index and create it again to use ElasticSearch, use CLI command for this (#2660)
```
Comment (raphj) [inline changelog.md:16]:
```suggestion
- ElasticSearch: Refactor of the indexing logic. It is necessary to drop the existing index and to create it again to use ElasticSearch, use the CLI command for this (#2660)
```

File: changelog.md [hunk 5]
```diff
@@ -1,5 +1,26 @@
...
+
+### Known issues
```
Comment (raphj) [inline changelog.md:18]:
```suggestion
### Known Issues
```

File: changelog.md [hunk 6]
```diff
@@ -1,5 +1,26 @@
...
+
+- SMTP server using implicit ssl can cause problem with Tracim. If you not activate `email.notification.smtp.use_implicit_ssl` and you use `email.processing_mode = sync`, when Tracim try to send email, backend error 500 appear after 5min and its also not possible to create new account.
```
Comment (raphj) [inline changelog.md:20]:
```suggestion
- SMTP servers using implicit SSL may cause issues with Tracim. If you not activate `email.notification.smtp.use_implicit_ssl` and you use `email.processing_mode = sync`, when Tracim tries to send email, 500 errors appear after a 5 minute timeout and its also not possible to create a new account.
```

File: changelog.md [hunk 7]
```diff
@@ -1,5 +1,26 @@
...
+In next release we add simple information [here](https://github.com/tracim/tracim/blob/master/backend/doc/setting.md#enabling-the-mail-notification-feature) to test your SMTP configuration with Tracim configuration file (issue #2827)
```
Comment (raphj) [inline changelog.md:21]:
```suggestion
In next release we will add information [here](https://github.com/tracim/tracim/blob/master/backend/doc/setting.md#enabling-the-mail-notification-feature) to test your SMTP configuration with the Tracim configuration file (issue #2827)
```

File: changelog.md [hunk 8]
```diff
@@ -1,5 +1,26 @@
...
+
+
 ## 2.6.2 / 2020-03-20
 
 ### Fixed issues
```
Comment (raphj) [inline changelog.md:26]:
```suggestion
### Fixed Issues
```

File: backend/doc/setting.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply backend/doc/setting.md]: Done

File: backend/doc/setting.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply backend/doc/setting.md]: Done

File: changelog.md (hunk 3 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

File: changelog.md (hunk 4 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

File: changelog.md (hunk 5 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done and change all `issues` to `Issues`

File: changelog.md (hunk 6 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

File: changelog.md (hunk 7 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

File: changelog.md (hunk 8 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done

**Pull request merged**